### ISSUE TITLE
How could I became a contributer of htsjdk?

### ISSUE THREAD

**Issue by renyaoxiang** [samtools/htsjdk#1035]
How could I became a contributer of htsjdk?

**Comment by yfarjoun** [comment]
Hi!

Great to hear that you'd like to help!

You could start by finding things you'd like to help with (https://github.com/samtools/htsjdk/issues), fork the repository and make changes on your fork. When you would like to submit your changes for review, open a pull request against your branch and someone will take a look at it and give comments. Once your branch is approved it will be merged into the master branch by someone with with the appropriate privileges.

Please feel free to ask more detailed questions if my answer is unclear or not what you were inquiring about.

Looking forward to your contributions!

**Comment by renyaoxiang** [comment]
Thank you.
